### PR TITLE
Removes duplicated code in PoPBenchmarker

### PR DESCRIPTION
**Pull request by iambmelt**
Benchmarker app cleanup.

The benchmarker app came _before_ any of this was written in MSAL (back during implementation discovery). Now that this code exists _for real_ inside of `common`, this benchmarker tool can depend directly on it. The overall quality of the benchmarker app leaves something to be desired, it's just enough to make it work.

Builds on work in:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1069

Blocked by:
~https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1176~
~https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1178~

Now unblocked by:
https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1178

Screenshot
image

### CODE REVIEW

File: pop-benchmarker/build.gradle [hunk 1]
```diff
@@ -21,5 +21,8 @@ android {
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$rootProject.ext.appCompatVersion"
-    implementation "com.nimbusds:nimbus-jose-jwt:7.8"
+    implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayoutVersion"
+    implementation(project(":common")) {
+        //transitive = false
```
Comment (iambmelt) [inline pop-benchmarker/build.gradle:26]: Remove / cleanup

**Comment by shahzaibj** [comment]
CI seems to be failing

**Comment by iambmelt** [comment]
> CI seems to be failing

Not sure what this failure has to do with these changes.... investigating

**Comment by iambmelt** [comment]
> > CI seems to be failing
> 
> Not sure what this failure has to do with these changes.... investigating

@shahzaibj looks to be related to this change:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1068

@AdamBJohnsonx Can you take a peek?

File: pop-benchmarker/build.gradle (hunk 1 above)
Comment (iambmelt) [reply pop-benchmarker/build.gradle]: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1068

**Comment by iambmelt** [comment]
> > > CI seems to be failing
> > 
> > 
> > Not sure what this failure has to do with these changes.... investigating
> 
> @shahzaibj looks to be related to this change:
> [AzureAD/microsoft-authentication-library-common-for-android#1068](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1068)
> 
> @AdamBJohnsonx Can you take a peek?

Per offline discussion, reverted those changes in:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1071

**Review by tanmaymanolkar1 — APPROVED** [review]

**Pull request merged**